components/eamxx/src/control/intensive_observation_period.cpp

```diff
@@ -693,6 +710,7 @@ read_iop_file_data (const util::TimeStamp& current_ts)
         const auto dx = iop_file_v_h(adjusted_file_levs-2) - iop_file_v_h(adjusted_file_levs-3);
         if (dx == 0) iop_file_v_h(adjusted_file_levs-1) = iop_file_v_h(adjusted_file_levs-2);
         else {
+          iop_file_pressure.sync_to_host();
```

Comment:
@bogensch For all IOP fields (where no surface value was defined in the IOP file), on GPU, the surface value will have been incorrect. Maybe this will positively affect results.